oesukam/andela-sendit
# #162570564  Improve on the project's documentation

#### What does this PR do?
Improves on the project's documentation

#### Description of Task to be completed?
- Add all endpoint to Swagger API file
- Add all endpoints to README.md
- Add LICENSE.md file

#### How should this be manually tested?
- Open `https://andela-sendit-api.herokuapp.com/api-docs` to see swagger api documentation with updated information

#### What are the relevant pivotal tracker stories?
`#162570564`
